packages/runtime/container-runtime/src/summary/summarizerTypes.ts

```diff
@@ -131,6 +131,9 @@ export interface IEnqueueSummarizeOptions extends IOnDemandSummarizeOptions {
 	readonly override?: boolean;
 }
 
+/** The stage up to which a summarize operation completed. */
```

Comment:
> up to which

So this stage is presumed to have failed if this info is included in an error?

Reply:
But it looks like the way it's used elsewhere (e.g. ContainerRuntime.submitSummary) is it lists the last stage that succeeded.  Now I see what "base" means - not even the "generate" step succeeded.


```suggestion
/** The stages of Summarize, used to describe how far progress succeeded in case of a failure at a later stage. */
```